base_name_search_improved/models/ir_model.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2016 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, tools
+
+# Extended name search is only used on some operators
+ALLOWED_OPS = set(['ilike', 'like'])
```

Comment:
What about uses sets definition directly instead of transform a list?
 - `ALLOWED_OPS = {'ilike', 'like'}`